language/tools/move-cli/tests/metatests/args.exp

```diff
@@ -7,10 +7,10 @@ Command `test dummy`:
 Command `test cov`:
```

Comment:
Should we expand this test to run multiple commands that hit multiple files to test the merging functionality?

Reply:
Yes, I am adding more test cases in an incoming commit.

Reply:
Tests for the merging functionality added in commit #fab6008